### PR TITLE
feat(15164): add forgot password on login page

### PR DESCRIPTION
**Pull request by Natallia-Harshunova**
https://kontur.fibery.io/Tasks/My-tasks-3575#Task/Add-Forgot-password-on-Kontur-apps-15164

image

## Summary by CodeRabbit

- **New Features**
  - Enhanced URL construction for registration and reset links in the login form.

- **Style**
  - Updated `forgotPasswordContainer` styling to use CSS variables and improved link color for better visual consistency.

### CODE REVIEW

File: src/features/user_profile/components/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -11,8 +11,13 @@ import type { ChangeEvent } from 'react';
...
 
-const registrationUrlEncoded = encodeURIComponent(`${configRepo.get().keycloakUrl}/realms/${configRepo.get().keycloakRealm}/protocol/openid-connect/registrations?client_id=account&response_type=code&scope=openid&redirect_uri=https://www.kontur.io/portfolio/`);
-const registrationUrl = `${configRepo.get().keycloakUrl}/realms/${configRepo.get().keycloakRealm}/protocol/openid-connect/logout?client_id=account&redirect_uri=` + registrationUrlEncoded;
+const registrationUrlEncoded = encodeURIComponent(
+  `${configRepo.get().keycloakUrl}/realms/${configRepo.get().keycloakRealm}/protocol/openid-connect/registrations?client_id=account&response_type=code&scope=openid&redirect_uri=https://www.kontur.io/portfolio/`,
+);
+const registrationUrl =
+  `${configRepo.get().keycloakUrl}/realms/${configRepo.get().keycloakRealm}/protocol/openid-connect/logout?client_id=account&redirect_uri=` +
+  registrationUrlEncoded;
+const resetUrl = `https://keycloak01.kontur.io/realms/kontur/login-actions/reset-credentials?client_id=account&tab_id=8cDrgjXYA-c`;
```
[the remark this answers is not in the document]
Comment (Komzpa) [reply src/features/user_profile/components/LoginForm/LoginForm.tsx]: @coderabbitai can you rebuild this URL based on configRepo.get().keycloakUrl and .keycloakRealm as above too?

[the remark this answers is not in the document]
Comment (Komzpa) [reply src/features/user_profile/components/LoginForm/LoginForm.tsx]: @coderabbitai what's tab_id for?

File: src/features/user_profile/components/LoginForm/LoginForm.module.css
Comment (albaranau) [inline src/features/user_profile/components/LoginForm/LoginForm.module.css:1]: Let's make font size on .link smaller (14px, like in Figma) and remove underline for unhovered `a`, that will look nicer

Comment (Komzpa) [reply src/features/user_profile/components/LoginForm/LoginForm.module.css:1]: @coderabbitai can you do it?

File: src/features/user_profile/components/LoginForm/LoginForm.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (albaranau) [reply src/features/user_profile/components/LoginForm/LoginForm.tsx]: @Natallia-Harshunova this should work:

> const resetUrl = `${configRepo.get().keycloakUrl}/realms/${configRepo.get().keycloakRealm}/login-actions/reset-credentials?client_id=account&redirect_uri=https://www.kontur.io/portfolio/`;

**Review by albaranau — APPROVED** [review]

**Pull request merged**